client/components/tinymce/index.jsx

```diff
@@ -178,6 +181,12 @@ module.exports = React.createClass( {
 
 		this.localize();
 
+		let toolbar1 = 'wpcom_add_media,';
+		if ( config.isEnabled( 'post-editor/contact-from' ) ) {
+			toolbar1 += 'wpcom_add_contact_form,';
```

Comment:
>there won't be a need for more feature flags.

What I had in mind was not exclusive to the contact form. We may have a need to expand upon this for other editor-related features to be added to the toolbar. I'm fine with addressing this when the time comes.